### DISCUSSION TITLE
How exactly to install it?

### DISCUSSION THREAD

**Discussion by niklaspoulsenSH** [firstof9/lovelace-grocy-meal-plan-card#9] [Q&A] [answered]
Hi, Sorry for an complete newbie question.
Would it be possible, that you could describe how i add this card?

Have it installed via HACS. But i dont know how to show it in frontend of my dashboard?

**Comment by firstof9** [comment, marked as answer]
https://github.com/firstof9/lovelace-grocy-meal-plan-card#card-config
